### PR TITLE
Expose official Sylius plugins 

### PR DESCRIPTION
**Pull request by Zales0123**
### Invoicing and Refund

Zrzut ekranu 2019-03-13 o 12 57 09

Zrzut ekranu 2019-03-13 o 12 57 36

Zrzut ekranu 2019-03-13 o 12 57 44

### Customer order cancellation and Customer Reorder

Zrzut ekranu 2019-03-13 o 13 06 30

### Admin order creation

Zrzut ekranu 2019-03-13 o 13 12 14

Zrzut ekranu 2019-03-13 o 13 12 33

Zrzut ekranu 2019-03-13 o 13 15 02

### RBAC

Zrzut ekranu 2019-03-13 o 13 28 11

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -80,8 +80,9 @@ before_script:
 script:
     - composer validate --strict
 
-    - echo "Testing (Behat - regular scenarios; ~@javascript && ~@todo && ~@cli)" "Sylius"
-    - vendor/bin/behat --strict --no-interaction -vvv -f progress --tags="~@javascript && ~@todo && ~@cli"
+# Temporarily disabling, in order to deal with plugins breaking tests
```
Comment (lchrusciel) [inline .travis.yml]: Is this PR a WIP, or do we want to turn off tests on Demo for now?

Comment (Zales0123) [reply .travis.yml]: We don't want to, but we have to 😢 All plugins together change **a lot** of Sylius logic, too much to handle it at once, especially in this PR.

Comment (lchrusciel) [reply .travis.yml]: Is there any chance, that we turn off just some of the features? Right now, we are not protected from the regression

Comment (Zales0123) [reply .travis.yml]: We can run SyliusDemo scenarios (I've just integrated them with SyliusRBACPlugin and will push it in a minute). For the Sylius scenarios - not doable right now, we would have to run each scenario and see does it fail and either exclude it or modify to make it green - there are just so many changes in Sylius structure provided in plugins. Still - running SyliusDemo scenarios is a good start, I think 🖖

File: src/Migrations/Version20180626065141.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20180626065141 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
```
Comment (lchrusciel) [inline src/Migrations/Version20180626065141.php]: What happened with these spaces? Same for all `DoctrineMigrations`.
```suggestion
    public function up(Schema $schema): void
```

**Comment by lchrusciel** [comment]
Thanks, Mateusz! :1st_place_medal:

**Comment by Zales0123** [comment]
🎉

**Comment by CoderMaggie** [comment]
💪

**Comment by lchrusciel** [comment]
🐺

**Pull request merged**